src/components/teacher/ConfirmationTables.tsx

```diff
@@ -157,9 +275,18 @@ export default function ConfirmationTables({ classroom }: { classroom: Classroom
 					<Tab label='One Time' />
 					<Tab label='Repeatable' />
 				</Tabs>
-				<Button sx={{ mb: 2 }} color='primary' onClick={() => handleConfirmAll()}>
-					Confirm All
-				</Button>
+				<Grid container columnSpacing={1} justifyContent='right' maxWidth={500}>
```

Comment:
Not sure how to use stack so that the buttons are the same size - when I change it to a Stack, the "Confirm GCR" button becomes a lot wider than "Confirm All"